.github/workflows/build.yml

```
@@ -36,8 +36,7 @@ jobs:
           name: coverage-${{ matrix.os }}
           path: coverage.*
 
-      - run: go install github.com/goreleaser/goreleaser/v2@latest
-      - run: goreleaser release 
+      - run: go tool goreleaser release 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)